An unexpected error has occurred: CalledProcessError: command: ('/home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/bin/python', u'/home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/bin/pip', 'install', '.')
return code: 1
expected return code: 0
stdout:
    Processing /home/master/.cache/pre-commit/repo0An3xc
    Collecting flake8
      Using cached flake8-3.7.9-py2.py3-none-any.whl (69 kB)
    Collecting ruamel.yaml>=0.15
      Using cached ruamel.yaml-0.16.10-py2.py3-none-any.whl (111 kB)
    Collecting toml
      Using cached toml-0.10.0-py2.py3-none-any.whl (25 kB)
    Collecting six
      Using cached six-1.14.0-py2.py3-none-any.whl (10 kB)
    Collecting typing
      Using cached typing-3.7.4.1-py2-none-any.whl (26 kB)
    Collecting enum34; python_version < "3.4"
      Using cached enum34-1.1.9-py2-none-any.whl (11 kB)
    Collecting entrypoints<0.4.0,>=0.3.0
      Using cached entrypoints-0.3-py2.py3-none-any.whl (11 kB)
    Processing /home/master/.cache/pip/wheels/c2/ea/a3/25af52265fad6418a74df0b8d9ca8b89e0b3735dbd4d0d3794/functools32-3.2.3.post2-py2-none-any.whl
    Collecting mccabe<0.7.0,>=0.6.0
      Using cached mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
    Collecting pyflakes<2.2.0,>=2.1.0
      Using cached pyflakes-2.1.1-py2.py3-none-any.whl (59 kB)
    Collecting configparser; python_version < "3.2"
      Using cached configparser-4.0.2-py2.py3-none-any.whl (22 kB)
    Collecting pycodestyle<2.6.0,>=2.5.0
      Using cached pycodestyle-2.5.0-py2.py3-none-any.whl (51 kB)
    Collecting ruamel.ordereddict; platform_python_implementation == "CPython" and python_version <= "2.7"
      Using cached ruamel.ordereddict-0.4.14.tar.gz (60 kB)
    Processing /home/master/.cache/pip/wheels/ba/ef/23/e4b121c0b4aaa5d5b222c1354e84caa3a111a2a51c87759a3e/ruamel.yaml.clib-0.2.0-py2-none-any.whl
    Building wheels for collected packages: pre-commit-hooks, ruamel.ordereddict
      Building wheel for pre-commit-hooks (setup.py): started
      Building wheel for pre-commit-hooks (setup.py): finished with status 'done'
      Created wheel for pre-commit-hooks: filename=pre_commit_hooks-2.3.0-py2.py3-none-any.whl size=35840 sha256=c8d931ed1e1d8a3dd029dd062697258da2510ac88224e0b0ae1e9540be311f8c
      Stored in directory: /tmp/pip-ephem-wheel-cache-FqTvzc/wheels/00/5a/b5/73443acbfce1f3c57bb62ee3ff31bee3b7727992e367e84eec
      Building wheel for ruamel.ordereddict (setup.py): started
      Building wheel for ruamel.ordereddict (setup.py): finished with status 'error'
      Running setup.py clean for ruamel.ordereddict
    Successfully built pre-commit-hooks
    Failed to build ruamel.ordereddict
    Installing collected packages: enum34, configparser, entrypoints, functools32, mccabe, pyflakes, typing, pycodestyle, flake8, ruamel.ordereddict, ruamel.yaml.clib, ruamel.yaml, toml, six, pre-commit-hooks
        Running setup.py install for ruamel.ordereddict: started
        Running setup.py install for ruamel.ordereddict: finished with status 'error'
    
stderr:
    DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
      ERROR: Command errored out with exit status 1:
       command: /home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-57X_XM/ruamel.ordereddict/setup.py'"'"'; __file__='"'"'/tmp/pip-install-57X_XM/ruamel.ordereddict/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-mrWSsp
           cwd: /tmp/pip-install-57X_XM/ruamel.ordereddict/
      Complete output (17 lines):
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-i686-2.7
      creating build/lib.linux-i686-2.7/ruamel
      creating build/lib.linux-i686-2.7/ruamel/ordereddict
      copying ./__init__.py -> build/lib.linux-i686-2.7/ruamel/ordereddict
      running build_ext
      building '_ordereddict' extension
      creating build/temp.linux-i686-2.7
      i686-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-eFYD24/python2.7-2.7.15~rc1=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/include/python2.7 -c ordereddict.c -o build/temp.linux-i686-2.7/ordereddict.o
      ordereddict.c:22:10: fatal error: Python.h: No such file or directory
       #include "Python.h"
                ^~~~~~~~~~
      compilation terminated.
      error: command 'i686-linux-gnu-gcc' failed with exit status 1
      ----------------------------------------
      ERROR: Failed building wheel for ruamel.ordereddict
        ERROR: Command errored out with exit status 1:
         command: /home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-57X_XM/ruamel.ordereddict/setup.py'"'"'; __file__='"'"'/tmp/pip-install-57X_XM/ruamel.ordereddict/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-JFD1qB/install-record.txt --single-version-externally-managed --compile --install-headers /home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/include/site/python2.7/ruamel.ordereddict
             cwd: /tmp/pip-install-57X_XM/ruamel.ordereddict/
        Complete output (17 lines):
        running install
        running build
        running build_py
        creating build
        creating build/lib.linux-i686-2.7
        creating build/lib.linux-i686-2.7/ruamel
        creating build/lib.linux-i686-2.7/ruamel/ordereddict
        copying ./__init__.py -> build/lib.linux-i686-2.7/ruamel/ordereddict
        running build_ext
        building '_ordereddict' extension
        creating build/temp.linux-i686-2.7
        i686-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-eFYD24/python2.7-2.7.15~rc1=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/include/python2.7 -c ordereddict.c -o build/temp.linux-i686-2.7/ordereddict.o
        ordereddict.c:22:10: fatal error: Python.h: No such file or directory
         #include "Python.h"
                  ^~~~~~~~~~
        compilation terminated.
        error: command 'i686-linux-gnu-gcc' failed with exit status 1
        ----------------------------------------
    ERROR: Command errored out with exit status 1: /home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-57X_XM/ruamel.ordereddict/setup.py'"'"'; __file__='"'"'/tmp/pip-install-57X_XM/ruamel.ordereddict/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-JFD1qB/install-record.txt --single-version-externally-managed --compile --install-headers /home/master/.cache/pre-commit/repo0An3xc/py_env-python2.7/include/site/python2.7/ruamel.ordereddict Check the logs for full command output.
    
Check the log at /home/master/.cache/pre-commit/pre-commit.log
